test/models/test_video_vqvae.py

```diff
@@ -87,6 +87,25 @@ def encoder(self, params):
         enc.eval()
         return enc
 
+    @pytest.fixture
+    def downsampler(self, params):
```

Comment:
Consider creating a single fixture to return a partial encoder instance with `params` exposed as the argument?